### ISSUE TITLE
Improve the whiteboard drawing experience by incorporating shortcut keys

### ISSUE THREAD

**Issue by tylercopeland** [bigbluebutton/bigbluebutton#5563]
To improve the drawing and interaction experience of the whiteboard, we can look at best practices found in design and digital drawing tools. For example, using shift to draw a straight line or using cmd and  [+] to zoom in. Having these shortcuts in place would give the user the flexibility of using one tool (pen) versus toggling between many (pan, pen, line). A great use case where this experience would be helpful is teaching math online. 

**Shortcuts**

Spacebar - Moving around the whiteboard when it's in a zoomed state
Shift - Straight line

_OSX_
Cmd + [ + ] - Zoom in  
Cmd + [ - ] - Zoom out
Cmd + [ 0 ] - Zoom 100%
Cmd + [ . ] - Zoom to fit
Cmd + Z  - Undo
Cmd + Shift + Z - Redo

_Windows_
Control + [ + ] - Zoom in  
Control + [ - ] - Zoom out
Control + [ 0 ] - Zoom 100%
Control + [ . ] - Zoom to fit
Control + Z  - Undo
Control + Shift + Z - Redo

**Comment by ffdixon** [comment]
Having, at a minimum, the ability to zoom in / zoom out with the keyboard would make it easy for someone to use a tablet (right hand) and zoom in / zoom out keyboard shortcut with left hand.

**Comment by pedrobmarin** [comment]
Extra suggestions from #9056:
```
Alt-Shift-T Text Tool
Alt-Shift-R Line Tool
Alt-Shift-W Pen Tool
```

**Comment by antobinary** [comment]
Shortcuts support was included in BBB 2.6 with the inclusion of tldraw
